omod/src/main/java/org/openmrs/module/registrationapp/page/controller/EditSectionPageController.java

```diff
@@ -234,10 +234,8 @@ private void updatePatientRelationships(Patient patient, String[] types, String[
                 
                 // Last character reveals relationship direction (aIsToB or bIsToA)
                 char relationshipDirection = types[i].charAt(types[i].length() - 1);                
-                if (relationshipDirection != 'A') {
-                	if (relationshipDirection != 'B') {
+                if (relationshipDirection != 'A' && relationshipDirection != 'B') {
                 		throw new APIException("Relationship direction not specified");
```

Comment:
```suggestion
                	throw new APIException("Relationship direction not specified");
```
You're missing one if nest, so everything inside has one less indent.

Reply:
Hey @sacull have merged the two if statements by an appendix of && operator. Since the && operator evaluates the second expression only if the first expression is true. So the nested if statement is inclusive in the suggested code.
                "if (relationshipDirection != 'A' && relationshipDirection != 'B') {
                		throw new APIException("Relationship direction not specified")"